changelog_unreleased/markdown/pr-9125.md

```diff
@@ -0,0 +1,15 @@
+#### Remove unnecessary blockquote `>` (#9125 by @shisama)
+
+<!-- prettier-ignore -->
+```js
+// Input
+> `x`
+> `y`
+
+// Prettier stable
+> `x` > `y`
+
+// Prettier master
+> `x` `y`
```

Comment:
This output is incorrect with our default options(`--prose-wrap=preserve `).
Can you change this to
```md
> `x`
> `y`
```
?

***

**Prettier pr-9125**
[Playground link](https://deploy-preview-9125--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEA+ABAAwB6YDpQaYCe+UIANCBAA4wCW0AzsqAIYBOHEA7gAqcELFGwA2PNsRZUARhzZgA1nBgBlGgvpQA5shgcArnCpwAtjLgATS1YAybHQbba4AMQgdTbGAx3IQbAYwEJQgABYwpqIA6mH08EwaYHCqQvH0AG7xxP5gTNIgWkxwHDB88tpeyABmYsVUAFZM2ABC8koqqmymcLZacDV1xiBN2Kpa2qJwAIoGEPCDovUgGhzFHP5eHIqWvORUNBxaMNH0ljBhyAAcAAwH3MXR8jT+h3DrGQNUAI5z8OW0YQBJgAWigcCsVlCHDgv3oMPKziqSFqS2GxVM9D0hnREyms3mAxRQyoMDYMlO50uSAATKT5PRRBMAMIQUzIkDvACsoQMxQAKuThKjlhkjABJKA2WCqMBHOgAQSlqhgxCmi2KAF9NUA)
```sh
--parser markdown
```

**Input:**
```markdown
> `x`
> `y`

```

**Output:**
```markdown
> `x`
> `y`

```